site/_docs/2022-04-13-valorem-options-litepaper.md

```diff
@@ -90,11 +90,11 @@ collateral into Valorem. They can specify the following:
 This information comprises a unique hash `keccak256(abi.encode(Option memory))`,
 which is then used to determine if that type of option already exists and, if 
 it doesn’t, create it. Once the option type is created, upon writing, the option
-writer will receive a `claim` token indicating that they have written the option.
-They will also receive an `option` token, indicating that they have the ability
-to exercise the option pursuant to the terms set during writing.  Both the
-`claim` token and the `option` token can be transferred or traded to other
-actors on the chain.
+writer will receive a `claim` token indicating that they have written the option,
+while the option owner will also receive an `option` token, indicating that they 
```

Comment:
The initial option owner is the claim writer.

Reply:
Ok, interesting. I'm trying to map this out to how options work in TradFi - perhaps that is where I'm going wrong, but the option writer sells the option to the option buyer. The options buyer then has the right to exercise the option (Euro vs American) at a specific point in time.

As the docs are written, once the option is written, 2 tokens are created: the `claim` token and the `option` token - both owned by the option writer. The act of buying the `option` leads to a transfer of ownership of the `option` token from the option writer to the option buyer. The `claim` token is still held by the option writer.

At this point, there are two potential outcomes:
1. The option buyer can exercise the `option` token, or 
2. The `option` token reaches expiry

Is that correct? 

I think I need to dive into the `option` and `claim` contracts to understand flow.